internal/daemon/controller/handlers/credentiallibraries/credentiallibrary_service.go

```diff
@@ -152,6 +167,10 @@ func (s Service) ListCredentialLibraries(ctx context.Context, req *pbs.ListCrede
 	if err != nil {
 		return nil, errors.Wrap(ctx, err, op)
 	}
+	genericRepo, err := s.baseCredentialLibraryRepoFn()
```

Comment:
Nit/question: should we change `genericRepo` to `baseRepo` to keep it consistent?

Reply:
doh, thanks